app/controllers/admin/variant_overrides_controller.rb

```diff
@@ -30,6 +29,22 @@ def bulk_update
       end
     end
 
+    def bulk_reset
+      collection_hash = Hash[params[:variant_overrides].each_with_index.map { |vo, i| [i, vo] }]
```

Comment:
Could these first four lines be replaced by a call to bulk_update above?
This would alleviate the repeated code that might mean one change needs to be made in two places in future.